packages/pd-manager/plugin.schema.json

```diff
@@ -0,0 +1,661 @@
+{
```

Comment:
this schema should be omitted because it gets generated, unless we have serialization issues, then we need to provide one, but disable the generation

Reply:
fixed